tests/Models/PV/test_pv_model.py

```diff
@@ -0,0 +1,319 @@
+import pytest
+import numpy as np
+import sys
+sys.path.append('src/illuminator/models/PV/')
+from pv_model import PV_py_model
```

Comment:
This is not necessary if the illuminator package is installed in editable mode... Test can import the package just as any other packages. e.g. `illuminator.models.PV.pv_model`

Reply:
This is an example: 

https://github.com/Illuminator-team/Illuminator/pull/65/files#diff-95369edfacb34bea06b151a80fa6864bbe239a274aadae6b403dad1fd001941cR12

Reply:
Done